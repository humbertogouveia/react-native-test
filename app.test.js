import {expect} from '@jest/globals'
import React from 'react'
import renderer from 'react-test-renderer'
import App from './App'

jest.mock('expo-status-bar', () => ({
    StatusBar: jest.fn(),
  }));
  

test('Tela deve ser renderizada corretamente',()=>{
    const app = renderer.create(<App />).toJSON()
    expect(app).toMatchSnapshot()
})