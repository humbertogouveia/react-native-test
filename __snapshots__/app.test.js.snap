// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tela deve ser renderizada corretamente 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Meu primeiro App React Native
  </Text>
</View>
`;
